airbyte-webapp/src/packages/cloud/views/auth/OAuthLogin/OAuthLogin.module.scss

```diff
@@ -45,6 +45,7 @@
 .github {
   background: #333;
   color: colors.$white;
+  border: none;
```

Comment:
:information_source: Get rid of the border of the OAuth buttons being dependent on CSS rule ordering.